src/layers/legacy/medCoreLegacy/gui/factories/medWorkspaceFactory.h

```diff
@@ -124,6 +109,21 @@ public slots:
                            medWorkspaceCreator creator,
                            medWorkspaceIsUsable isUsable = nullptr, bool isActive = true);
 
+
+    QHash<QString, medWorkspaceFactory::Details *> workspaceDetails() const;
+    QList<medWorkspaceFactory::Details *> workspaceDetailsSortedByName(bool activeOnly = false) const;
+    medWorkspaceFactory::Details * workspaceDetailsFromId(QString identifier) const;
+
+    bool isUsable(QString identifier) const;
+
+    QList<QString> workspacesFromCategory(const QString& category) const;
+
+public slots:
+
+    medAbstractWorkspaceLegacy* createWorkspace(QString type,QWidget* parent=0);
```

Comment:
Since you are modifying the fill you could put parent to nullptr instead of 0

Reply:
github is displaying this all wrong. I didn't touch these lines of code, I moved the function that was below to put it above.
Also I know it would be a minimal change but by principal I don't like changing code that isn't the target of the PR.